### PR TITLE
Function to create brain mask

### PR DESCRIPTION
**Pull request by jsjol**
Refers to #8 .

Basically a thin wrapper of dipy.segment.mask.median_otsu. Currently uses default settings, after #9 is resolved we should check if these settings work well for HCP data.

### CODE REVIEW

**Comment by jsjol** [comment]
Is the test that the masked brain volume is of a certain size a good fit for a unit test? Is it more of a functional test? 

A problem is that it requires realistic data, which makes the test run in approx 15 s, which is way too long for a unit test. Still I think it is valuable to do a similar test somewhere.

File: test_createBrainMask.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import unittest
+from os.path import join, expanduser, isdir
+import numpy as np
+from dipy.data.fetcher import fetch_scil_b0, read_siemens_scil_b0
+from createBrainMask import createBrainMaskFromb0Data
+
+
+class TestCreateBrainMask(unittest.TestCase):
+
+    def test_createBrainMaskFromb0Data(self):
```
Comment (marbank) [inline test_createBrainMask.py]: Add a function description to function and of function arguments.

File: createBrainMask.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+
+from dipy.segment.mask import median_otsu
+
+
+def createBrainMaskFromb0Data(b0Data):
+    medianRadius = 4
+    numIterations = 4
```
Comment (mariabankestad) [inline createBrainMask.py]: Add a function description to function and of function arguments. Why MedianRadius = 4?

Comment (mariabankestad) [reply createBrainMask.py]: Added description in new commit. MedianRadius = 4 is the default, hence it unnecessary to define, so I removed it.

**Comment by jsjol** [comment]
Closes #8

**Pull request merged**